Delete the old version3.21

meishenlieshou opened the pull request:
Delete the old version 3.21, otherwise V3.3 will not be available on the marketplace
Thanks for merging！

pamval commented:
There are a lot of changes in your commit which are unrelated to your plugin. It looks as if your repository wasn't up to date, maybe?

Can you rebase to the latest master and then remove version 3.2.1 from there?

Thanks!

meishenlieshou commented:
In the previous pull requests, I did rebase to the latest master and then modified, which always resulted in irrelevant content being affected, so I guessed it was due to my local edit tool format typographic differences.

For the last time 3 days ago, I modified it based on the lastest master directly on the github page and pulled request again. It seems OK this time. Could you help to review it again.

thank you very much!

> There are a lot of changes in your commit which are unrelated to your plugin. It looks as if your repository wasn't up to date, maybe?
> 
> Can you rebase to the latest master and then remove version 3.2.1 from there?
> 
> Thanks!

meishenlieshou commented:
Hi，Pedro Vale：

In the previous pull requests, I did rebase to the latest master and then
modified, which always resulted in irrelevant content being affected, so I
guessed it was due to my local edit tool format typographic differences.

For the last time 3 days ago, I modified it based on the lastest master
directly on the github page and pulled request again. It seems OK this
time. Could you help to review it again.

thank you very much!

Pedro Vale <notifications@github.com> 于2020年7月17日周五 下午5:25写道：

> There are a lot of changes in your commit which are unrelated to your
> plugin. It looks as if your repository wasn't up to date, maybe?
>
> Can you rebase to the latest master and then remove version 3.2.1 from
> there?
>
> Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pentaho/marketplace-metadata/pull/739#issuecomment-659991126>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA6BIZIAMQGCFUN4GWMM2LLR4AKI3ANCNFSM4O5DWX4A>
> .
>
